packages/wonder-blocks-form/src/components/text-field.tsx

```diff
@@ -247,12 +249,15 @@ class TextField extends React.Component<PropsWithForwardRef, State> {
                         ? styles.disabled
                         : this.state.focused
                         ? [styles.focused, light && styles.defaultLight]
-                        : this.state.error && [
+                        : !!this.state.error && [
                               styles.error,
                               light && styles.errorLight,
                           ],
                     style && style,
-                ])}
+                    // Cast `this.state.error` into boolean since it's being
+                    // used as a conditional
+                    !!this.state.error && styles.error,
```

Comment:
I don't see this in the originally logic.  Was there an issue with the render of the error state that necessitated this change?